app/models/job.ts

```diff
@@ -11,6 +11,9 @@ import Record from './record';
 import { Transaction } from '../util/db';
 import JobLink, { getLinksForJob, JobLinkOrRecord } from './job-link';
 
+// how long data generated by this job will be available
+export const EXPIRATION_DAYS = 30;
```

Comment:
Any reason to not make this an environment variable? Are you thinking there might be an issue if it's set differently from when the job was first run?

Reply:
I considered making it an environment variable, but rejected it for these reasons
1. The AC says "hard coded to 30 days"
2. The 30 days is being enforced by a lifecycle policy no the bucket, not by code that would read an environment variable
3. We already have too many environment variables so I didn't want to add another without good reason